Resources/Locale/en-US/cargo/bounties.ftl

```diff
@@ -100,3 +117,20 @@ bounty-description-warm-cloth = The Unath construction crew freezes and is unabl
 bounty-description-battery = As the Arachnid settlement prepares for a solar flare, they are requesting a large shipment of power batteries. We're sending out a request for delivery.
 bounty-description-lasergun = The Salvage Caravan requests a large shipment of laser weapons to mop up a hive of xenomorphs.
 bounty-description-food = After the rat king invasion, a neighboring unathi station was left completely without food. A large meat food shipment is needed.
+bounty-description-fruit = Remilia helped the chaplain catch a troublemaker hiding in the chapel, and the crew wants to reward him for his good work.
```

Comment:
I think remilia should be a unique creature with a name, not duplicated on other stations at the same point in time? If you imagine these are orders from other stations, things should be a bit different there

Reply:
Good point! Swapped it over to a monkey.